openvdb/doc/examplecode.txt

```diff
@@ -2060,4 +2060,57 @@ for (auto leafIter = points->tree().beginLeaf(); leafIter; ++leafIter) {
 }
 @endcode
 
+@section sPointMove Moving Points in Space
+
+As points are stored within voxels in an implicit spatially organised data structure, moving points in space requires re-sorting the data.
```

Comment:
(minor) as suggested by @toddkeeler , the term "re-bucketing" might be more appropriate than "re-sorting".

Reply:
@fgochez Yes, re-bucketing is probably more correct, but I think I might just add both terms here for additional clarity. :)